lib/raven/processor.rb

```diff
@@ -2,11 +2,8 @@
 
 module Raven
   class Processor
-    attr_accessor :sanitize_fields
```

Comment:
whether or not I merge this, I'm going to do this change in a separate PR. this stuff should not be in the processor base class.